larky/pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>com.verygood.security</groupId>
     <artifactId>larky</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>urlparse01</version>
```

Comment:
mistake?

Reply:
yes, this is not ready to be merged, just to share with drew for progress update. I will clean up the code after at the end